package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "dependencies": {
+    "css-loader": "^6.7.1",
+    "html-webpack-plugin": "^5.5.0",
+    "style-loader": "^3.3.1",
+    "webpack": "^5.74.0",
+    "webpack-cli": "^4.10.0",
+    "webpack-dev-server": "^4.10.0"
+  },
+  "name": "leaderboard",
+  "description": "",
+  "version": "1.0.0",
+  "main": "index.js",
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.34.0",
+    "@typescript-eslint/parser": "^5.34.0",
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-react": "^7.30.1",
+    "hint": "^7.1.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack --mode development",
+    "start": "webpack serve --open"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/melashu/Leaderboard.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/melashu/Leaderboard/issues"
+  },
+  "homepage": "https://github.com/melashu/Leaderboard#readme"
+}
```

Comment:
- [ ] It is a requirement that scripts like build, start are added in package.json. See the example below.
````json
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
````

Reply:
hi @munsa1 

Thank you for your constructive feedback. However, I haven't seen the requirement to use the scripts like build, and start added in package.json as you mention. when I use your suggestion `npm run start` can not open the browser. It simply watches the change.

Reply:
Here is the requirement 

![image](https://user-images.githubusercontent.com/30173722/186422729-9d0e9e28-7423-453c-8fac-571bfe6f27ff.png)